### DISCUSSION TITLE
Common Issues thread - suggest additions here!

### DISCUSSION THREAD

**Discussion by mdcfe** [EssentialsX/Essentials#3955] [General]
This discussions thread is for people to suggest common issues to add to the pinned Common Issues thread.

**Note: this thread is not for EssentialsX support.** If you need help, consider [searching the Q&A forum](https://github.com/EssentialsX/Essentials/discussions/categories/q-a) or asking on [Discord](https://discord.gg/casfFyh)

**Comment by JasonHorkles** [comment]
How do I disable this? (Picture of motd and the show mail on join message)
